eldritch/test_events.py

```diff
@@ -262,6 +296,151 @@ def testMoveMultipleSpaces(self):
     self.assertEqual(self.char.place.name, "Graveyard")
     self.assertEqual(self.char.movement_points, 1)
 
+  def testIllegalMoveMultipleSpaces(self):
+    self.char.movement_points = 1
+    movement = Sequence(
+        [MoveOne(self.char, dest) for dest in ["Easttown", "Rivertown", "Graveyard"]],
+        self.char,
+    )
+    self.assertFalse(movement.is_resolved())
+    self.assertEqual(self.char.movement_points, 1)
+
+    self.state.event_stack.append(movement)
+    self.resolve_until_done()
+
+    self.assertTrue(movement.is_resolved())
+    self.assertEqual(self.char.place.name, "Easttown")
+    self.assertEqual(self.char.movement_points, 0)
+
+  def testMoveMultipleThroughMonsterFight(self):
```

Comment:
Is the key difference between this test and testMoveOneSpaceToMonster that this one starts with the character moving multiple spaces? Maybe add a comment to this test.